src/zope/component/tests/test_hookable.py

```diff
@@ -68,62 +68,38 @@ def bar():
     def test_cant_assign_original(self):
         from zope.component.hookable import hookable
         def foo():
-            pass
+            raise NotImplementedError()
```

Comment:
Random drive-by comment: to me `raise AssertionError('should not be called')` seems like the best way to spell must-not-be-called, and `raise NotImplementedError('please override in subclasses')` is the best way to spell abstract-method.